things/database.py

```diff
@@ -425,191 +425,6 @@ def was_modified_today(self):
         todays_date = datetime.datetime.now().date()
         return last_modified_date >= todays_date
 
-    # -------- Historical methods (TK: transform) --------
-
-    def get_trashed(self):
```

Comment:
However, getting trashed tasks would be of interest #16

Reply:
Agreed 👍. Want to give it a try? I will add some remarks to #16 as for how.